src/Symfony/Bridge/Twig/composer.json

```diff
@@ -22,7 +22,7 @@
     "require-dev": {
         "symfony/asset": "~2.7",
         "symfony/finder": "~2.3",
-        "symfony/form": "~2.7.8|~2.8,>2.8-BETA1",
+        "symfony/form": "~2.7,>=2.7.8",
```

Comment:
Lets use `^2.7.8`. I know we're not using caret so far. But changing all ~ is probably not worth considering all the conflicts. But we can upgrade gradually when it makes sense.